gatsby-config.js

```diff
@@ -65,12 +66,23 @@ const sourceToPluginConfig = {
 const externalSourcePlugins = () => {
   const sourcePlugins = [];
 
-  if (!process.env.SKIP_SOURCING && gracefulFs.existsSync(sourceFilename)) {
-    console.log(
-      `${ANSI_BLUE}###### Sourcing from ${sourceFilename} #######${ANSI_STOP}`,
-    );
+  if (!process.env.SKIP_SOURCING) {
+    // default to full set of sources
+    let sources = Object.keys(sourceToPluginConfig).reduce((result, source) => {
+      result[source] = true;
+      return result;
+    }, {});
+
+    if (gracefulFs.existsSync(sourceFilename)) {
+      console.log(
+        `${ANSI_BLUE}###### Sourcing from ${sourceFilename} #######${ANSI_STOP}`,
+      );
+      console.log(
+        `${ANSI_GREEN}Note that ${sourceFilename} is no longer strictly required - the full set of docs will be loaded in its absence.${ANSI_STOP}`,
```

Comment:
Maybe be a bit more prescriptive here: the intention is to give folks a heads-up that they might need to take action to get the sources that they need, so might as well be explicit about that action. 
```suggestion
        `${ANSI_GREEN}Note: ${sourceFilename} is no longer required; delete it to load the full set of docs.${ANSI_STOP}`,
```